src/events/VoiceState.ts

```
@@ -6,6 +6,8 @@ import { composeDashboard } from "../utils/dashboard"
 import { isInvited, removeInvite } from "../utils/inviteStatus"
 import { blacklistUsers } from "../utils/misc"
 
+import { Prisma } from "../generated/prisma/client"
```

ESLint

[error] Unable to resolve path to module '../generated/prisma/client'.

(import-x/no-unresolved)